app/views/homes/index.html.erb

```diff
@@ -1,4 +1,5 @@
-<p>
-  Modify me in `app/views/homes/index.html.erb` or change the root directive in
-  `config/routes.rb`
-</p>
+
+
+<div id="app">
```

Comment:
Whitespace lines 1 and 2 should be deleted. the empty line between the div can be deleted as well